src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -350,7 +351,8 @@ private void doWithZkUtils(final ZkUtilsAction action) throws Exception {
         }
     }
 
-    private Integer calculateKafkaPartitionCount(final EventTypeStatistics stat) {
+    @VisibleForTesting
```

Comment:
I'm such a big hater of this annotation :)
But we can discuss if we want that or not in Nakadi in one of our meetings.
For now it's ok for me.